src/Integration.UnitTests/Helpers/ToolWindowServiceTests.cs

```diff
@@ -97,9 +99,10 @@ public void Show_FindWindowSucceededAndNonNullToolWindow_ToolWindowShown()
             var testSubject = new ToolWindowService(serviceProviderMock.Object);
 
             // Act
-            testSubject.Show(ValidToolWindowId);
+            testSubject.Show(ValidToolWindowId, shouldActivateWindow);
 
-            frameMock.Verify(x => x.Show(), Times.Once);
+            var expectedShowCallCount = shouldActivateWindow ? 1 : 0;
+            frameMock.Verify(x => x.Show(), Times.Exactly(expectedShowCallCount));
```

Comment:
Modified test to check `frame.Show` is called correctly.
All of the other test changes are to just to fix up the mock to have the new parameter and to check that all of the existing callers are passing `true`.